x-hooks/use-dao.tsx

```diff
@@ -64,19 +64,23 @@ export function useDao() {
    * Change network to a known address if not the same
    * @param networkAddress
    */
-  function changeNetwork() {
-    const { networkAddress, chain_id } = state.Service?.network?.active || {};
-    
-    if (!state.Service?.active || !networkAddress || !chain || state.spinners.switchingChain)
+  function changeNetwork(chainId = '', address = '') {
+    const networkAddress = address || state.Service?.network?.active?.networkAddress;
+    const chain_id = chainId || state.Service?.network?.active?.chain_id;
+
+    if (!state.Service?.active || !networkAddress || !chain_id || state.spinners.switchingChain)
       return;
 
     if (state.Service?.active?.network?.contractAddress === networkAddress)
       return;
 
     const service = state.Service.active;
 
-    if (+chain_id !== +chain.chainId || chain.chainRpc !== state.Service?.active?.web3Host)
-      return;
+    if(chain){
```

Comment:
?

Reply:
it was necessary because when directly accessing "/create-bounty" it loaded the first chain, but when changing to another chain it did not allow me to load the new DAO.